cmd/root_test.go

```diff
@@ -286,3 +230,13 @@ func fileExists(name string) (bool, error) {
 		return false, err
 	}
 }
+
+func setup() {
+	mockServer = fixture.StartHttpTargetTestServer(8080, []fixture.PathResponseHandler{}, false)
```

Comment:
Minor: what about `mockHttpServer` instead of `mockServer`, to be more clearly opposed to `mockGrpcServer`?